acq4/modules/DataManager/DataManagerModule.py

```diff
@@ -112,14 +106,22 @@ def setLogDir(self):
             d = d.parent()
         self.manager.setLogDir(d)
 
+    def loadPinnedImages(self):
+        cam_mod = self.manager.getModule("Camera")
+        current_dir = self.manager.getCurrentDir()
+        for f in current_dir.ls():
+            if f.endswith('.tif'):
+                f = current_dir[f]
+                frame = Frame(f.read(), f.info().deepcopy())
```

Comment:
Frame.load(fileHandle) automatically checks for linked files?

Reply:
`Frame.load` isn't a thing that exists... Maybe you are suggesting I make that? `Frame` isn't tied to the `FileHandle.read` architecture, but it totally could be, if you want.